src/content/eol/2024/04/eol-04-24-24-polling-api.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "End of life for Azure SQL database integration polling API"
+summary: "Azure is deprecating the SDK the polling API uses and will no longer be providing critical updates to this API, creating security concerns moving forward."
+publishDate: '2024-04-22'
+eolEffectiveDate: '2024-12-01'
+---
+
+On December 1, 2024, New Relic will end support for the Azure SQL database integration polling API, because Azure is deprecating the SDK that the polling API uses. Only customers who capture data with the polling API will experience a change in their data if they’re using the Azure SQL Database integration.  
```

Comment:
```suggestion
On December 1, 2024, New Relic will end support for the Azure SQL database integration polling API, because Azure is deprecating the SDK that the polling API uses. You don't need to take action unless you capture data with the polling API and you're using the Azure SQL Database integration.  
```

I found this second sentence quite confusing, and attempted a rewrite. We should try to confirm this rewrite with the relevant product team in case I changed the meaning.

Reply:
Much clearer rewrite, thanks! I'll check in with the product team

Reply:
I've confirmed this with other messages I've found in product channels on Slack